runtests.py

```diff
@@ -34,7 +34,7 @@ def run_pytest(run_slow=False, extra_args=None):
     # Be sure to ignore subrepos
     pytest_args = ['-vv', '-rw', '--durations=10', '--ignore=./external-deps',
                    '-W ignore::UserWarning', '--timeout=120',
-                   '--timeout_method=thread']
+                   '--timeout_method=thread', '-x']
```

Comment:
```suggestion
                   '--timeout_method=thread']
```

This shouldn't be necessary now that this PR is ready for review, right?